ActivityLauncherApp/src/main/res/values/strings.xml

```diff
@@ -50,4 +50,6 @@
     <string name="theme_default">System Default</string>
     <string name="theme_light">Light Theme</string>
     <string name="theme_dark">Dark Theme</string>
+    <string name="private_activities_summary">Hide private (unexported and disabled) activites, that cannot be started without root access</string>
```

Comment:
Please place below related setting and rename to settings_description_hide_private

Reply:
Or summary